.travis.yml

```diff
@@ -45,6 +45,7 @@ before_script:
 
 
 script:
+  - make shellcheck
```

Comment:
@seanmalloy What would you think, better to keep this here separately, or to include it as part of the general `make test` target, in Makefile l.55? https://github.com/KohlsTechnology/eunomia/blob/612a4769ae0dd8bd0ab15a936dd0fae49dfafd3a/Makefile#L55